sycl/test-e2e/Matrix/element_wise_all_ops_int8_packed_impl.hpp

```diff
@@ -95,6 +99,31 @@ int main() {
           }
         }
       });
+}
+
+int main() {
+  queue q;
+  std::vector<combination> combinations =
+      q.get_device()
+          .get_info<sycl::ext::oneapi::experimental::info::device::
+                        matrix_combinations>();
+
+  for (unsigned int i = 0; i < combinations.size(); i++) {
+    if (combinations[i].nsize == 0) {                        // Intel AMX
+      test<int8_t, int, /*TK*/ 64, /*TN*/ 16, /*VNNI*/ 4>(); // should work
```

Comment:
Could you clarify, what do you mean by comment "should work"?
Also, pls, do not interleave comment with the code, as it is more difficult to maintain that way.
My preference is to keep comments either one-line if you want them next to the code:
` code; // comment`
 or have a block of comment not interleaved with the source code, like this:
```
// first comment line
// second comment line
code;
```
In this particular case, it seems you can do just this:
```suggestion
    if (combinations[i].nsize == 0) { // Intel AMX
      test<int8_t, int, /*TK*/ 64, /*TN*/ 16, /*VNNI*/ 4>();
```

Reply:
This was a typo left over from my debugging efforts. Will remove